app/controllers/relationships_controller.rb

```diff
@@ -4,6 +4,8 @@ class RelationshipsController < ApplicationController
   def create
     @user = User.find(params[:followed_id])
     current_user.follow(@user)
+    relationship = Relationship.find_by(follower_id: current_user, followed_id: @user)
+    relationship.relationship_activity(current_user)
```

Comment:
@ivanraybuglosa 
You can use the `active_relationships` association to get `relationship` object.
And the `build_activity` method, made along with the `has_one` association

```
relationship = user.active_relationships.find_by(followed: other_user)
relationship.build_activity(user: user)
```